rfc-028-adopt-a-code-of-conduct.md

```diff
@@ -100,13 +100,13 @@ time.
 
 ## Next steps
 
-- Set up an email contact code-of-conduct@dxw.com that goes to Dominic Baggott
-- Set up an email contact code-of-conduct-md@dxw.com that goes to David Mann
-- Confirm that Dave and Dom are happy to take this role
-- Apply the guidance to our existing GitHub projects
-- Add ourselves to the list of Contributor Covenant adopters – see
-  [guidance](https://github.com/ContributorCovenant/contributor_covenant#adding-a-project-to-the-list-of-adopters)
-- Create fallback CONTRIBUTING.md and CODE_OF_CONDUCT.md files for the dxw
+- [ ] Set up an email contact code-of-conduct@dxw.com that goes to Dominic Baggott; see [#148](https://github.com/dxw/tech-team-rfcs/issues/148)
```

Comment:
Who should this be now that Dom has left? (I realise this wasn't part of the change but it should be flagged)

Reply:
It will be bob and F, see #148

Reply:
I wasn't sure whether to change it in the RFC, but decided it's best to leave it as it was for "as-written" accuracy. We wouldn't want to come back and change it every time, after all.

Reply:
Agree @Floppy